src/main.c

```diff
@@ -1654,7 +1628,7 @@ static void cmdline_handler(int argc, char **argv)
 	g_free(text);
 
 done:
-	g_clear_pointer(&context, g_option_context_free);;
+	return;
```

Comment:
Why the return statement here at the very end of a void function?

Reply:
So I can keep the "done:" label and not have to change all the "goto done;"s in this function in this patch.